### DISCUSSION TITLE
Dynamic Combo Chart

### DISCUSSION THREAD

**Discussion by LunaUrsa** [TripSit/drugs#143] [Development]
Currently, we have the combo chart here: https://combo.tripsit.me/

This website is built off of the https://github.com/TripSit/comboapp repo

One of our ultimate goals for the future is to create an interface for the combo chart that allows people to select whichever drugs they want, and they can receive a tripsit-branded chart that they can print out and display.

In order to do that we need to update or just completely redo our https://github.com/TripSit/combogen/  code, which is currently built in python

A previous colleague (Ivan) was able to create a proof of concept going in NextJS:  https://github.com/TripSit/combogen/tree/next-js

We were in contact with the https://psychcombo.com/  team and in talks about merging their dynamic table generator with our website, as well as combining our drug database, but it's been a while since we heard from them

**Comment by raphaelsaunier** [comment]
Would love to help with this later this year. In the meantime, feel free to borrow any ideas from what I shared in #394. 😉
